ts/features/messages/hooks/useAttachmentDownload.tsx

```diff
@@ -2,17 +2,55 @@ import { useCallback, useEffect, useState } from "react";
 import * as pot from "@pagopa/ts-commons/lib/pot";
 import ReactNativeBlobUtil from "react-native-blob-util";
 import RNFS from "react-native-fs";
+import { identity, pipe } from "fp-ts/lib/function";
+import * as E from "fp-ts/lib/Either";
+import * as O from "fp-ts/lib/Option";
+import * as TE from "fp-ts/lib/TaskEither";
 import i18n from "../../../i18n";
 import { mixpanelTrack } from "../../../mixpanel";
 import { useIODispatch, useIOSelector } from "../../../store/hooks";
 import { ContentTypeValues } from "../../../types/contentType";
-import { isIos } from "../../../utils/platform";
+import { isAndroid } from "../../../utils/platform";
 import { showToast } from "../../../utils/showToast";
 import { downloadAttachment } from "../../../store/actions/messages";
 import { UIAttachment } from "../../../store/reducers/entities/messages/types";
 import { downloadPotForMessageAttachmentSelector } from "../../../store/reducers/entities/messages/downloads";
 import { trackThirdPartyMessageAttachmentShowPreview } from "../../../utils/analytics";
 
+const taskCopyToMediaStore = (
+  { displayName, contentType }: UIAttachment,
+  path: string
+) =>
+  TE.tryCatch(
+    () =>
+      ReactNativeBlobUtil.MediaCollection.copyToMediaStore(
+        {
+          name: displayName,
+          parentFolder: "",
+          mimeType: contentType
+        },
+        "Download",
```

Comment:
Is this a placeholder or a string showed to the user? In this case what do you think about use a locale string? `i18n.t("")`

Reply:
The `Download` is a `MediaType` so we can avoid using `i18n.t("")`